eslint/includes/typescript.yaml

```diff
@@ -7,3 +7,8 @@ rules:
     '@typescript-eslint/no-empty-interface': off
     '@typescript-eslint/no-explicit-any': off
     '@typescript-eslint/no-non-null-assertion': off
+
+    '@typescript-eslint/no-inferrable-types':
+        - warn
+        -   ignoreParameters: true
+            ignoreProperties: true
```

Comment:
the CS looks a bit weird. Is that correct?

Reply:
Now it's a bit better.

It still looks weird, but that is YAML, you could also write:

```yaml
        -   
            ignoreParameters: true
            ignoreProperties: true
```

or 

```yaml
- {ignoreParameters: true, ignoreProperties: true}
```